_RFCs/0001-rfc-for-rfcs.md

```diff
@@ -0,0 +1,185 @@
+
+Modifications to "active" RFCs can be done in follow-up pull requests. We strive to write each RFC in a manner that it will reflect the final design of the feature; but the nature of the process means that we cannot expect every merged RFC to actually reflect what the end result will be.
+
+In general, once accepted, RFCs should not be substantially changed. Only very minor changes should be submitted as amendments. More substantial changes should be new RFCs, with a note added to the original RFC. Exactly what counts as a "very minor change" is up to the core-plans maintainers to decide.
+
+## Reviewing RFCs
+[Reviewing RFCs]: #reviewing-rfcs
+
+When the RFC pull request is first opened, a core plans maintainer will be assigned as a shepherd to the RFC to ensure it makes as much progress through the RFC pipeline as is possible. While the RFC PR is open the assigned shepherd may schedule meetings  with the author and/or relevant stakeholders to discuss the issues in greater detail, and in some cases the topic may be discussed at a maintainers meeting. In either case a summary from the meeting will be posted back to the RFC pull request.
+
+Maintainers make the final decisions about RFCs after the benefits and drawbacks are well understood. These decisions can be made at any time, but the maintainers will regularly issue decisions. When a decision is made, the RFC pull request will either be merged or closed. In either case, if the reasoning is not clear from the discussion in thread, the maintainers will add a comment describing the rationale for the decision.
+
+
+## Implementing an RFC
+[Implementing an RFC]: #implementing-an-rfc
+
+Some accepted RFCs represent vital changes that need to be implemented right away. Other accepted RFCs can represent changes that can wait until some arbitrary contributor feels like doing the work. Every accepted RFC has an associated issue tracking its implementation in the Core-Plans repository; thus that associated issue can be assigned a priority via the triage process that the team uses for all issues in the Core-Plans repository.
+
+The author of an RFC is not obligated to implement it. Of course, the RFC author (like any other contributor) is welcome to post an implementation for review after the RFC has been accepted. With that in mind, the best chance for an RFC to get implemented quickly is much more likely to happen if the RFC author either intends to write the feature themselves, **OR** has had conversations with a maintainer who is advocating for the RFC and has expressed the willingness/availability to implement the change in the case that the RFC is accepted.
+
+If you are interested in working on the implementation for an "active" RFC, but cannot determine if someone else is already working on it, feel free to ask (e.g. by leaving a comment on the associated issue).
+
+
+## RFC Postponement
+[RFC Postponement]: #rfc-postponement
```

Comment:
Unless I missed something here, there's nothing explicit in this process for future evaluation of postponed RFCs.  I have some concerns about accruing postponed RFCs and ambiguity around what that means for further discussion or submission of competing RFCs.

What's the rationale for wanting to postpone vs just outright rejecting with a final comment to possibly revisit/reopen later?

Reply:
You are correct: "Postponed pull requests may be re-opened when the time is right. We don't have any formal process for that, you should ask maintainers in the relevant slack channel." I don't know that we want to have a formal process for it. I think this is more about having a safety valve in the case that theres something we think is reasonable, but that just realistically can't do or wont be able to do for a protracted period of time. I imagine these like large changes that are exceptionally breaking in nature but perhaps the _right thing_ :tm: .

Reply:
To phrase this another way.  Can we re-open rejected RFCs in the event of new information or a changed opinion from core maintainers?  Because if we can and there is no process for reviewing postponed RFCs, it doesn't really seem like a different category from my perspective.

Reply:
There is no system in place for re-opening rejected RFCs in this draft i don't believe.